lib/wrappers/openssl.nim

```diff
@@ -402,13 +430,40 @@ else:
     let theProc = cast[proc() {.cdecl.}](sslSymNullable("OPENSSL_add_all_algorithms_conf"))
     if not theProc.isNil: theProc()
 
+  proc SSL_CTX_set_options*(ctx: SslCtx, op: clong): clong =
+    let theProc {.global.} = cast[proc(ctx: SslCtx, op: clong): clong {.cdecl, gcsafe.}](sslSymNullable("SSL_CTX_set_options"))
+    if not theProc.isNil:
+      theProc(ctx, op)
+    else:
+      SSL_CTX_ctrl(ctx, SSL_CTRL_OPTIONS, op, nil)
+
+  proc SSL_CTX_clear_options*(ctx: SslCtx, op: clong): clong =
+    let theProc {.global.} = cast[proc(ctx: SslCtx, op: clong): clong {.cdecl, gcsafe.}](sslSymNullable("SSL_CTX_clear_options"))
+    if not theProc.isNil:
+      theProc(ctx, op)
+    else:
+      SSL_CTX_ctrl(ctx, SSL_CTRL_CLEAR_OPTIONS, op, nil)
+
   proc getOpenSSLVersion*(): culong =
     ## Return OpenSSL version as unsigned long or 0 if not available
     let theProc = cast[proc(): culong {.cdecl, gcsafe.}](utilSymNullable("OpenSSL_version_num", "SSLeay"))
     result =
       if theProc.isNil: 0.culong
       else: theProc()
 
+  proc SSL_CTX_set_min_proto_version*(ctx: SslCtx, version: cint): cint =
+    ## Set the minimum supported protocol version.
+    const MainProc = "SSL_CTX_set_min_proto_version"
+    let theProc {.global.} = cast[proc(ctx: SslCtx, version: cint): cint {.cdecl, gcsafe.}](sslSymNullable(MainProc))
```

Comment:
These `.global` procs are bad style, prefer real globals instead. Also: You probably want `.raises: [], locks: 0`

Reply:
Wouldn't that mean I'd have to produce an unique global name for every proc?